ts/boot/scheduleLocalNotifications.ts

```diff
@@ -44,7 +43,24 @@ export const scheduleLocalNotificationsAccessSpid = () => {
  * Remove all the local notifications relating to authentication with spid
  */
 export const removeScheduledNotificationAccessSpid = () => {
-  PushNotification.cancelLocalNotifications({
-    id: FIRST_ACCESS_SPID_TAG
-  });
+  /**
+   * With the current library version seems that cancelLocalNotifications doesn't work.
+   *
+   * eg. example code that doesn't work:
+   *
+   *   PushNotification.getScheduledLocalNotifications(x =>
+   * x
+   * .filter(notification => notification.data.tag === FIRST_ACCESS_SPID_TAG)
+   * .forEach(firstAccessNotification => {
+   *      PushNotification.cancelLocalNotifications({
+   *        id: firstAccessNotification.id.toString()
+   *      });
+   *    })
+   * );
+   *
+   * At the moment the "first access spid" is the only kind of scheduled notification and for this reason
+   * is safe to use PushNotification.cancelAllLocalNotifications();
```

Comment:
```suggestion
   * it is safe to use PushNotification.cancelAllLocalNotifications();
```

Reply:
updated in 28775d7145c985e6d52f51289f91abe6bb2a1ef1